docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,506 @@
+# Create Your First Tizen Addon
+
+**Welcome to Tizen Addon development!**
+
+An addon is a JavaScript snippet with a manifest file that allows additional common functionalities such as pop-up to Web applications.
+
+Study the following instructions to help familiarize yourself with the Tizen [Web application development process](../../tutorials/process/app-dev-process.md) as well as using Tizen Studio and installing the created application on the emulator or target device. The addon is not a Web application, but it is very similar to the Web application in developing process. With the following instructions, you can create and run a basic addon, which gets timings of various application life-cycle:
```

Comment:
Please add "the" before "developing process".

Reply:
done